### PR TITLE
YOLayout for NSView

### PR DESCRIPTION
**Pull request by gabriel**

### CODE REVIEW

**Comment by gabriel** [comment]
NSVIEW!!! Fuck yeah no more AppKit layout bullshit :)

File: YOLayout.podspec [hunk 1]
```diff
@@ -1,5 +1,5 @@
 #
-#  Be sure to run `pod spec lint YOLayout.podspec' to ensure this is a
+#  Be sure to run `pod spec lint YOLayout.podspec" to ensure this is a
```
Comment (johnboiles) [inline YOLayout.podspec]: ?

File: YOLayout/YONSView.h [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+   [super viewInit];
+   NSView *subview = [[NSView alloc] init];
+   [self addSubview:subview];
+   self.viewLayout = [YOLayout layoutWithLayoutBlock:^(id<YOLayout> layout, CGSize size) {
+     [layout setFrame:CGRectMake(0, 0, size.width, 30) view:subview];
+     return CGSizeMake(size.width, 30);
+   }];
+ }
+ */
+@interface YONSView : NSView { }
```
Comment (johnboiles) [inline YOLayout/YONSView.h]: Necessary to have the `{ }`?

**Comment by johnboiles** [comment]
This is awesome!!

File: YOLayout.podspec (hunk 1 above)
Comment (gabriel) [inline YOLayout.podspec]: oops reverting

File: YOLayout/YONSView.h (hunk 2 above)
Comment (gabriel) [inline YOLayout/YONSView.h]: Fixed

**Comment by johnboiles** [comment]
:ship: it

**Pull request merged**